pkg/sql/export/exportcsv.go

```diff
@@ -178,6 +178,11 @@ func (sp *csvWriter) Run(ctx context.Context, output execinfra.RowReceiver) {
 		alloc := &tree.DatumAlloc{}
 
 		writer := newCSVExporter(sp.spec)
+		if sp.spec.HeaderRow != nil && *sp.spec.HeaderRow {
+			if err := writer.Write(sp.spec.ColNames); err != nil {
```

Comment:
can you test this with double-quoted-named columns? eg `"a"`, `"A"`, `"a""B"`, etc